src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2187,6 +2190,10 @@ bool AdaptDispatch::SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle)
         fEnableBlinking = false;
         actualType = CursorType::VerticalBar;
         break;
+
+    default:
+        // Invalid argument should be ignored.
+        return true;
```

Comment:
I think it's fair to treat parameter larger than 6 as a "success". I mean, why bother sending it to the connected terminal when we know it should be ignored.

Reply:
Well now that you mention it, technically we should probably be passing through any unknown parameters. The connected terminal is not necessarily WT, and it could potentially support values larger than 6. We were actually just discussing this in issue https://github.com/microsoft/terminal/issues/7382#issuecomment-683318287. I don't think it's a big deal, but returning false here is probably better (assuming that doesn't cause any other problems).

Reply:
That makes sense. I wasn't really sure about it. Let's try 'false' for this.